.travis.yml

```diff
@@ -14,6 +14,7 @@ install:
   - pip install ".[all]"
 
 script:
+  - for i in {1..10}; do pytest -vs pydra; done;
```

Comment:
I know it might be annoying, but I'd just keep it for some time when working on submitter and workers. Only because some issues happen not as often...

Reply:
```suggestion
  - for i in {1..10}; do pytest -vs pydra/engine/tests/test_workflow.py; done;
```